lib/bugsnag/delayed_job.rb

```diff
@@ -33,7 +33,8 @@ def error(job, error)
               p[:id]           = payload.id           if payload.respond_to?(:id)
               p[:display_name] = payload.display_name if payload.respond_to?(:display_name)
               p[:method_name]  = payload.method_name  if payload.respond_to?(:method_name)
-              p[:args]         = payload.args         if payload.respond_to?(:args)
+              p[:args]         = payload.respond_to?(:args) ? payload.args : payload.to_h
```

Comment:
@kattrali It's available as a method on the `Struct` class which is passed to `Delayed::Job#enqueue`. This outputs the parameters to that job in the form of a hash.

However I can see it might raise problems where the user has a Custom class that does not support this method, so I'll update it to not add `args` in such case. What do you think?